vignettes/epidist.Rmd

```diff
@@ -1,201 +1,250 @@
+Important examples include:
+
+* the incubation period (time between infection and symptom onset),
+* serial interval (time between symptom onset of infectee and symptom onset of infected), and 
+* generation interval (time between infection of infectee and infection of infected).
+
+Unfortunately, estimating delays accurately from observational data is usually difficult.
+In our experience, the two main issues are:
+
+1. interval censoring, and
+2. right truncation.
 
-Simulate data from an outbreak.
+Don't worry if you've not come across these terms before.
+In Section \@ref(data), we will explain what they mean by simulating data like that we might observe during an ongoing infectious disease outbreak.
+Next, in Section \@ref(fit), we show how `epidist` can be used to estimate delays using a statistical model which properly accounts for these two issues.
+Finally, in Section \@ref(compare), we demonstrate that the fitted delay distribution accurately recovers the underlying truth.
 
-```{r simulate-outbreak}
+If you would like more technical details, the `epidist` package implements models following best practices as described in @park2024estimating and @charniga2024best.
+
+# Example data {#data}
+
+Data should be formatted as a [`data.table`](https://cran.r-project.org/web/packages/data.table/index.html) with the following columns for use within the `epidist` package:
+
+* `case`: 
+* `ptime`:
+* `stime`:
+
+Here we simulate data in this format, and in doing so explain the two main issues with observational delay data.
+<!-- Please see as of yet unwritten vignette for information about how to transform your data of other formats to the right format. -->
+
+First, we use the [Gillepsie algorithm](https://en.wikipedia.org/wiki/Gillespie_algorithm) to generate infectious disease outbreak data (Figure \@ref(fig:outbreak)) from a stochastic compartmental model:
+
+```{r}
 outbreak <- simulate_gillespie(seed = 101)
 ```
 
-Define the secondary distribution to use in the simulation
+(ref:outbreak) Early on in the epidemic, there is a high rate of growth in new cases. As more people are infected, the rate of growth slows. (Only every 50th case is shown to avoid over-plotting.)
 
-```{r secondary-dist}
-secondary_dist <- data.table(
-  meanlog = 1.8, sdlog = 0.5
-) |>
+```{r outbreak, fig.cap="(ref:outbreak)"}
+outbreak[case %% 50 == 0, ] |>
+  ggplot(aes(x = ptime, y = case)) +
+  geom_point(col = "#56B4E9") +
+  labs(x = "Primary event time (day)", y = "Case number") +
+  theme_minimal()
+```
+
+`outbreak` is a [`data.table`](https://cran.r-project.org/web/packages/data.table/index.html) with the columns `case` and `ptime`.
+
+Now, to generate secondary events, we will use a lognormal distribution (Figure \@ref(fig:lognormal)) for the delay between primary and secondary events:
+
+```{r}
+secondary_dist <- data.table(meanlog = 1.8, sdlog = 0.5) |>
   add_natural_scale_mean_sd()
```

Comment:
Note there may be some work spinning up on making a small package to carry these transforms around